pmm-admin_test.go

```diff
@@ -402,6 +403,40 @@ Get http://xyz/qan-api/ping: dial tcp: lookup xyz.*: no such host
 	assertRegexpLines(t, expected, string(output))
 }
 
+func testConfigServerHideCredentials(t *testing.T, data pmmAdminData) {
+	defer func() {
+		err := os.RemoveAll(data.rootDir)
+		assert.Nil(t, err)
+	}()
+
+	err := os.MkdirAll(data.rootDir+pmm.PMMBaseDir, 0777)
+	assert.Nil(t, err)
+
+	cmd := exec.Command(
+		data.bin,
+		"config",
+		"--server",
+		fmt.Sprintf("%s:%s", "172.0.0.1", "8080"),
+		"--server-user",
+		"test",
+		"--server-password",
+		"123",
+	)
+
+	output, err := cmd.CombinedOutput()
+	assert.Error(t, err)
```

Comment:
Can we use a more specific check there?

Reply:
any suggestions?

Reply:
What error does it return? `*os.ExitError`? Then we can check the type and exit code.